src/experimental/test/property/arbitraries/canister_methods/inspect_message_method_arb.ts

```diff
@@ -6,93 +6,50 @@ import { Test } from '../../test';
 import { BoolArb } from '../candid/primitive/bool_arb';
 import { Context } from '../types';
 import { UniqueIdentifierArb } from '../unique_identifier_arb';
-import {
-    BodyGenerator,
-    CanisterMethodConstraints,
-    generateMethodImplementation,
-    MethodImplementationLocationArb,
-    TestsGenerator
-} from '.';
+import { BodyGenerator, generateMethodImplementation, TestsGenerator } from '.';
 
 export type InspectMessageMethod = {
     imports: Set<string>;
-    globalDeclarations: string[];
     sourceCode: string;
     tests: Test[][];
 };
 
 export function InspectMessageMethodArb(
-    context: Context<CanisterMethodConstraints>,
+    context: Context,
     generator: {
         generateBody: BodyGenerator;
         generateTests: TestsGenerator;
     }
 ): fc.Arbitrary<InspectMessageMethod> {
-    const api = context.api;
-    const constraints = context.constraints;
     return fc
         .tuple(
             UniqueIdentifierArb('canisterProperties', 'property'),
             BoolArb({ ...context, constraints: {} }),
-            MethodImplementationLocationArb,
             UniqueIdentifierArb('globalNames')
         )
-        .map(
-            ([
-                functionName,
-                returnType,
-                defaultMethodImplementationLocation,
-                methodName
-            ]): InspectMessageMethod => {
-                const methodImplementationLocation =
-                    api === 'class'
-                        ? 'INLINE'
-                        : (constraints.methodImplementationLocation ??
-                          defaultMethodImplementationLocation);
-
-                const imports = new Set(['inspectMessage']);
+        .map(([functionName, returnType]): InspectMessageMethod => {
+            const imports = new Set(['inspectMessage']);
 
-                const methodImplementation = generateMethodImplementation(
-                    [],
-                    returnType,
-                    generator.generateBody,
-                    methodImplementationLocation,
-                    functionName,
-                    methodName,
-                    api,
-                    api === 'class'
-                );
-
-                const globalDeclarations =
-                    methodImplementationLocation === 'STANDALONE'
-                        ? [methodImplementation]
-                        : [];
+            const methodImplementation = generateMethodImplementation(
+                [],
+                returnType,
+                generator.generateBody,
+                functionName,
+                true
```

Comment:
What is this parameter for?

Reply:
it seems like that parameter indicates that this method is the inspect message method